addons/project_todo/static/tests/todo_done_checkmark.test.js

```diff
@@ -0,0 +1,186 @@
+import { test, expect, describe } from "@odoo/hoot";
+import { queryFirst, queryAll, queryOne } from "@odoo/hoot-dom";
+import { animationFrame } from "@odoo/hoot-mock";
+import { click } from "@mail/../tests/mail_test_helpers";
+import { mountView, onRpc, mountWithCleanup, getService, contains } from "@web/../tests/web_test_helpers";
+import { defineTodoModels } from "./todo_mock_models";
+import { WebClient } from "@web/webclient/webclient";
+
+defineTodoModels();
+describe.current.tags("desktop");
+
+test("Check clicking on todo_done_checkmark in kanban view with initial state is in progress", async () => {
+    expect.assertions(5);
+    onRpc("web_save", (args) => {
+        expect(args.args[1].state).toBe("1_done", {
+            message: "The task should be marked as done",
+        });
+        expect.step("web_save")
+    });
+
+    await mountView({
+        resModel: "project.task",
+        type: "kanban",
+        actionMenus: {}
```

Comment:
```suggestion
        actionMenus: {},
```

Is it necessary to pass actionMenus as empty?